docker-compose.yml

```diff
@@ -1,88 +1,88 @@
+        ports:
+            - '3306'
+        healthcheck:
+            test: [ "CMD", "healthcheck.sh", "--connect", "--innodb_initialized" ]
+            start_period: 10s
+            interval: 10s
+            timeout: 5s
+            retries: 3
+        environment:
+            - MYSQL_ROOT_PASSWORD=password
+            - MYSQL_USER=db
+            - MYSQL_PASSWORD=db
+            - MYSQL_DATABASE=db
+            #- ENCRYPT=1 # Uncomment to enable database encryption.
 
-  phpfpm:
-    image: itkdev/php8.3-fpm:latest
-    user: ${COMPOSE_USER:-deploy}
-    networks:
-      - app
-      - frontend
-    extra_hosts:
-      - "host.docker.internal:host-gateway"
-    environment:
-      - PHP_XDEBUG_MODE=${PHP_XDEBUG_MODE:-off}
-      - PHP_MAX_EXECUTION_TIME=30
-      - PHP_MEMORY_LIMIT=256M
-      # Depending on the setup, you may have to remove --read-envelope-from from msmtp (cf. https://marlam.de/msmtp/msmtp.html) or use SMTP to send mail
-      - PHP_SENDMAIL_PATH=/usr/bin/msmtp --host=mail --port=1025 --read-recipients --read-envelope-from
-      - DOCKER_HOST_DOMAIN=${COMPOSE_DOMAIN}
-      - PHP_IDE_CONFIG=serverName=localhost
-    depends_on:
-      mariadb:
-        condition: service_healthy
-    volumes:
-      - .:/app
+    phpfpm:
+        image: itkdev/php8.3-fpm:latest
+        user: ${COMPOSE_USER:-deploy}
+        networks:
+            - app
+            - frontend
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
+        environment:
+            - PHP_XDEBUG_MODE=${PHP_XDEBUG_MODE:-off}
+            - PHP_MAX_EXECUTION_TIME=30
+            - PHP_MEMORY_LIMIT=256M
+            # Depending on the setup, you may have to remove --read-envelope-from from msmtp (cf. https://marlam.de/msmtp/msmtp.html) or use SMTP to send mail
+            - PHP_SENDMAIL_PATH=/usr/bin/msmtp --host=mail --port=1025 --read-recipients --read-envelope-from
+            - DOCKER_HOST_DOMAIN=${COMPOSE_DOMAIN}
+            - PHP_IDE_CONFIG=serverName=localhost
+        depends_on:
+            mariadb:
+                condition: service_healthy
+        volumes:
+            - .:/app
 
-  nginx:
-    image: nginxinc/nginx-unprivileged:alpine
-    networks:
-      - app
-      - frontend
-    depends_on:
-      - phpfpm
-    ports:
-      - '8080'
-    volumes:
-      - ./.docker/templates:/etc/nginx/templates:ro
-      - .:/app
-    environment:
-      NGINX_FPM_SERVICE: ${COMPOSE_PROJECT_NAME}-phpfpm-1:9000
-      NGINX_WEB_ROOT: /app/public
-      NGINX_PORT: 8080
-      NGINX_MAX_BODY_SIZE: 5M
-    labels:
-      - "traefik.enable=true"
-      - "traefik.docker.network=frontend"
-      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.rule=Host(`${COMPOSE_DOMAIN}`)"
-      # HTTPS config - uncomment to enable redirect from :80 to :443
-      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
-      - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
+    nginx:
```

Comment:
The templates use 2 space indents.
Please run

itkdev-docker-compose template:install

to install the latest template files.